app/components/security_key_image_component.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class SecurityKeyImageComponent < BaseComponent
+  def initialize(mobile:)
```

Comment:
Could we add `**tag_options` to apply to the wrapper element, to follow convention with other components?

```suggestion
  def initialize(mobile:, **tag_options)
```

Reply:
Sure, done in 5f01b121e9278c6a5b6d93fd32da39c31a139d84 with custom `data-` and `aria-`  support just in case 😉